data/mods/ssb/moves.ts

```diff
@@ -2573,6 +2573,38 @@ export const Moves: {[k: string]: ModdedMoveData & {gen?: number}} = {
 		type: "Normal",
 	},
 
+	// naziel
+	notsoworthypirouette: {
+		accuracy: 100,
+		basePower: 0,
+		category: "Status",
+		desc: "55% chance to OHKO the target; otherwise, it OHKOs itself.",
+		shortDesc: "55% chance to OHKO target. 45% to OHKO user.",
+		name: "Not-so-worthy Pirouette",
+		isNonstandard: "Custom",
+		pp: 5,
+		priority: 0,
+		flags: {},
+		onTryMove() {
+			this.attrLastMove('[still]');
+		},
+		onPrepareHit(target, source) {
+			this.add('-anim', source, "High Jump Kick", target);
+		},
+		onHit(target, source) {
+			let koed: Pokemon;
+			if (this.random() < 0.55) {
+				koed = target;
+			} else {
+				koed = source;
+			}
+			koed.faint();
```

Comment:
```suggestion
			if (this.random() < 0.55) {
				target.faint();
			} else {
				source.faint();
			}
```

Reply:
You could golf this even more with something like `(this.random () > 0.55 ? target : source).faint();` but I think my suggestion is more readable.